lib/Runtime/Library/Chakra.Runtime.Library.vcxproj

```diff
@@ -34,6 +34,7 @@
         $(MSBuildThisFileDirectory)..\..\Runtime\ByteCode;
         $(MSBuildThisFileDirectory)..\..\JITClient;
         $(ChakraJITIDLIntDir);
+        $(IcuIncludeDir)\common; <!-- // TODO cleanup: Is this needed here in Runtime.Library? -->
```

Comment:
I was not sure whether other directories under this path would be needed. I'll come back to this and refactor when the work is complete.